Recrutify/Recrutify.DataAccess/Repositories/CandidateRepository.cs

```diff
@@ -143,6 +143,16 @@ public Task UpdateStatusByIdsAsync(IEnumerable<Guid> ids, Guid projectId, Status
             return GetCollection().UpdateManyAsync(filter, updateDefinition, updateOptions);
         }
 
+        public Task UpdateIsAssignedAsync(IEnumerable<InterviewAppointmentSlot> interviewAppointmentSlots, Guid projectId)
```

Comment:
Why you pass to candidate repository appointments ?? You should pass only what is related to Candidate model